service/api_logic/login_logic.py

```diff
@@ -0,0 +1,111 @@
+from flask import current_app, url_for
+from flask_mail import Message
+from itsdangerous import URLSafeTimedSerializer
+from dto.api_output import OutPutUser
+from database.models import User
+from exept.handle_exeptions import handle_exceptions
+import bcrypt
+from logger.logger import Logger
+
+api_logic_logger = Logger("api_logic_logger", "api_logic_logger.log")
```

Comment:
We are in BusinessLayer, so I don't see any reason which would justify this being named "api_logic_logger". Just "log" or "logger" will suffice